src/Categories/Double/Core.agda

```diff
@@ -0,0 +1,297 @@
+{-# OPTIONS --without-K --safe #-}
+module Categories.Double.Core where
+
+open import Level
+
+open import Relation.Binary using (Rel; IsEquivalence; Setoid; REL)
+open import Relation.Binary.PropositionalEquality as ≡ using (_≡_)
+import Relation.Binary.Reasoning.Setoid as SetoidR
+
+import Categories.Category.Unbundled as Cat
+
+-- Equality of squares in a double category
+module _ {o ℓ e : Level} {Obj : Set o} (Hor Ver : Cat.Category Obj ℓ e) where
```

Comment:
I'm pretty sure that we can usefully have both categories live at different levels, so that this would involve 5 levels instead of 3.